client/ionic/src/pages/LocalMaps.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {
+  IonContent,
+  IonHeader,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  IonItem,
+} from '@ionic/react';
+
+class LocalMaps extends React.Component<{}> {
```

Comment:
I think we should stick only with functional components, and no classes (and remove the one that exists in Triage) @epicfaace what do you think React-wise

Reply:
Yeah, ideally it would be good to use functional components all around and use React Hooks when we need to use state. These components (`LocalMaps`) and (`ProtectYourself`) don't use state at all so it would make most sense and involve minimal effort to convert them to functional components.

Reply:
Fine with me, I'm new to react so I did it in `Triage` because it was the simplest way for me to add state.

Reply:
agree with sticking with functional components. if you're interested, Dan Abramov has [a great explanation](https://overreacted.io/how-are-function-components-different-from-classes/) of why!